.github/workflows/update-snapshot.yml

```diff
@@ -0,0 +1,91 @@
+name: Update to new data snapshot (regenerate colors and sharing images).
+
+on:
+  repository_dispatch:
+    types: update-data-snapshot
+
+env:
+  AWS_S3_BUCKET: content.covidactnow.org
+  BRANCH: ${{ github.event.client_payload.branch }}
+  SNAPSHOT_ID: ${{ github.event.client_payload.snapshot_id }}
+  # Used in the shared images URL and in the PR branch.
+  RUN_ID: ${{ github.event.client_payload.snapshot_id }}-${{github.run_number}}
+
+jobs:
+  update-data-snapshot:
+    runs-on: ubuntu-latest
+    steps:
+      # Setup
+      - name: Verify snapshot provided
+        if: ${{ !env.SNAPSHOT_ID }}
+        run: 'echo "Missing client_payload parameter: snapshot_id" ; exit 1'
+      - name: Verify branch provided
+        if: ${{ !env.BRANCH }}
+        run: 'echo "Missing client_payload parameter: branch" ; exit 1'
+      - uses: actions/checkout@v2
+        with:
+          repository: covid-projections/covid-projections
+          ref: '${{ env.BRANCH }}'
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      # Update snapshot URL and regenerate map colors.
+      - run: yarn install
+      - run: 'echo "{ \"data_url\": \"https://data.covidactnow.org/snapshot/${{env.SNAPSHOT_ID}}/\" }" | python -mjson.tool > src/assets/data/data_url.json'
+      - run: yarn update-calculated-interventions
+
+      # Generate Share Images and Upload to S3
+      - run: yarn serve -l 3000 &
+      - run: npx wait-on -t 120000 http://localhost:3000/
+      - run: yarn generate-share-images
+      - uses: jakejarvis/s3-sync-action@master
+        with:
+          args: '--acl public-read --follow-symlinks --delete'
+        env:
+          #TODO(michael): Make default secrets work.
```

Comment:
Working with Simon to get this figured out.  Right now we have two different AWS keys for our different jobs, which isn't ideal.